src/qt/dbb_gui.cpp

```diff
@@ -2081,63 +2081,97 @@ void DBBDaemonGui::updateTransactionTable(DBBWallet *wallet, bool historyAvailab
+                iconName = ":/icons/tx_" + QString::fromStdString(actionUV.get_str());
+            QStandardItem *item = new QStandardItem(QIcon(iconName), QString::fromStdString(DBB::formatMoney(amountUV.get_int64())));
             item->setToolTip(tr("Double-click for more details"));
+            item->setFont(font);
+            item->setTextAlignment(Qt::AlignRight | Qt::AlignVCenter);
             transactionTableModel->setItem(cnt, 1, item);
         }
 
-        UniValue amountUV = find_value(obj, "amount");
-        if (amountUV.isNum())
+        UniValue addressUV = find_value(obj["outputs"][0], "address");
+        if (addressUV.isStr())
         {
-            QStandardItem *item = new QStandardItem(QString::fromStdString(DBB::formatMoney(amountUV.get_int64())));
+            QStandardItem *item = new QStandardItem(QString::fromStdString(addressUV.get_str()));
             item->setToolTip(tr("Double-click for more details"));
+            item->setFont(font);
+            item->setTextAlignment(Qt::AlignCenter); 
             transactionTableModel->setItem(cnt, 2, item);
         }
-
-        UniValue feeUV = find_value(obj, "fees");
-        if (feeUV.isNum())
+        
+        UniValue timeUV = find_value(obj, "time");
+        if (timeUV.isNum())
         {
-            QStandardItem *item = new QStandardItem(QString::fromStdString(DBB::formatMoney(feeUV.get_int64())));
+            QDateTime timestamp;
+            timestamp.setTime_t(timeUV.get_int64());
+            QStandardItem *item = new QStandardItem(timestamp.toString(Qt::SystemLocaleShortDate));
             item->setToolTip(tr("Double-click for more details"));
+            item->setFont(font);
             transactionTableModel->setItem(cnt, 3, item);
         }
+            
+        UniValue confirmsUV = find_value(obj, "confirmations");
+        {
+            QString iconName;
+            QString tooltip;
+            if (confirmsUV.isNum())
+            {
+                tooltip = QString::number(confirmsUV.get_int());
+                if (confirmsUV.get_int() > 5)
+                    iconName = ":/icons/confirm6";
+                else
+                    iconName = ":/icons/confirm" + QString::number(confirmsUV.get_int());
+            } else {
+                tooltip = "0";
+                iconName = ":/icons/confirm0";
+            }
+            QStandardItem *item = new QStandardItem(QIcon(iconName), "");
+            item->setToolTip(tooltip + tr(" confirmations"));
+            item->setTextAlignment(Qt::AlignCenter); 
+            transactionTableModel->setItem(cnt, 4, item);
+        }
+
         UniValue txidUV = find_value(obj, "txid");
         if (txidUV.isStr())
         {
             QStandardItem *item = new QStandardItem(QString::fromStdString(txidUV.get_str()) );
-            transactionTableModel->setItem(cnt, 4, item);
+            transactionTableModel->setItem(cnt, 0, item);
         }
 
         cnt++;
     }
+  
+    if (!cnt) {
```

Comment:
I agree. After playing with it a while, I agree it would be better to show an empty table. I'll update the PR.